### PR TITLE
MAUI-4293-UG document content for .NET MAUI SfMaskedEntry control

### PR DESCRIPTION
**Pull request by AarthiArjunan**
Description:
Here I have added the Overview, Migration, Formatting Value, Culture, Getting Started and Mask Type content in .NET MAUI.

### CODE REVIEW

**Comment by AarthiArjunan** [comment]
rebuild

**Comment by AarthiArjunan** [comment]
rebuild

**Comment by AarthiArjunan** [comment]
rebuild

File: MAUI/Masked-Entry/Migration.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfMaskedEntry | Syncfusion 
+description: Learn all about Migrating from Syncfusion Xamarin MaskedEdit to Syncfusion .NET MAUI MaskedEntry control and more here.
+platform: maui
+control: SfMaskedEntry
+documentation: ug
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Migration.md:7]: Documentation: UG

File: MAUI/Masked-Entry/Migration.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+---  
+
+# Migrating from Xamarin SfMaskedEdit to .NET MAUI SfMaskedEntry
+
+To migrate easier from `Xamarin SfMaskedEdit` to `.NET MAUI SfMaskedEntry,` most of the APIs were kept from Xamarin SfMaskedEdit in MAUI SfMaskedEntry. However, to maintain the consistency of API naming in MAUI SfMaskedEntry, we renamed some of the APIs. The APIs changed in MAUI SfMaskedEntry from Xamarin SfMaskedEdit are detailed as follows:
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Migration.md:12]: To migrate easier from the `Xamarin SfMaskedEdit` to `.NET MAUI SfMaskedEntry,` most of the APIs are kept from the Xamarin SfMaskedEdit in MAUI SfMaskedEntry. However, to maintain the consistency of API naming in MAUI SfMaskedEntry, rename some of the APIs. The APIs changed in MAUI SfMaskedEntry from Xamarin SfMaskedEdit are detailed as follows.

File: MAUI/Masked-Entry/Overview.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Overview.md:2]: Layout: Post

File: MAUI/Masked-Entry/Overview.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+title: About .NET MAUI MaskedEntry control | Syncfusion
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Overview.md:3]: Title: About .NET MAUI MaskedEntry control | Syncfusion

File: MAUI/Masked-Entry/Overview.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+description: Learn all about the introduction of Syncfusion MAUI MaskedEntry (SfMaskedEntry) control with essential features and more here.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Overview.md:4]: Description: Learn about the introduction of Syncfusion MAUI MaskedEntry (SfMaskedEntry) control with essential features and more.

File: MAUI/Masked-Entry/Overview.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+platform: maui
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Overview.md:5]: Platform: Maui

File: MAUI/Masked-Entry/Overview.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+control: SfMaskedEntry
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Overview.md:6]: Control: SfMaskedEntry

File: MAUI/Masked-Entry/Overview.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Overview.md:7]: Documentation: UG

File: MAUI/Masked-Entry/Overview.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+# .NET MAUI MaskedEntry Overview
+
+The `.NET MAUI MaskedEntry` is an advanced version of the input control that restricts input of certain characters, text, and numbers by using a mask pattern. This control is used to create a template for providing information such as telephone numbers, email IDs, IP addresses, product keys, and so on.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Overview.md:12]: The `.NET MAUI MaskedEntry` is an advanced version of the input control that restricts input of certain characters, text, and numbers by using a mask pattern. This control creates a template for providing information such as telephone numbers, email IDs, IP addresses, product keys, and more.

File: MAUI/Masked-Entry/Overview.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+
+![MaskedEntry control overview in MAUI](MaskedEntry_Images/maui_maskedentry_overview.png)
+
+## Key features
+
+* `MaskTypes`: The input can be masked with a fixed or variable length by setting the mask type to simple or regex, respectively.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Overview.md:18]: * `MaskTypes`: The input can be masked with a fixed or variable length by setting the mask type to simple or regex.

**Comment by anandhan-rajagopal** [comment]
Changes are fine

**Review by PaulAndersonS — APPROVED** [review]

**Pull request merged**